news/2024/2024-12-22-touhou-project-mania-cup-4th-registrations-now-open.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: "Touhou Project Mania Cup 4th: Registrations Now Open!"
+date: 2024-12-22 00:00:00 +0000
+---
+
+![](/wiki/Tournaments/TMC/4th/image/banner@2x.png)
+
+With the celebration of the Touhou Project's 30th birthday, the Touhou Project Mania Cup is returning for its 4th iteration, bringing tons of improvement in competitiveness and player enjoyment!
+
+The [Touhou Project Mania Cup 4th](https://osu.ppy.sh/community/forums/topics/2015815), or THMC 4 for short, is a team-based osu!mania 4K tournament exclusively featuring Touhou-related songs. This tournament aims to provide top-tier competition to the tournament scene.
+
+We've had 80 teams [last year](/wiki/Tournaments/TMC/3rd), where team **Bloxy cola** reached the top with an undefeated performance. [This year](/wiki/Tournaments/TMC/4th), with more teams joining the competition, which team will rise to the challenge and become the best? 
+
+## What's new
+
+- **A Group Stage will be introduced in this year**, featuring 16 groups of 4 teams each. This means that we will have **64 teams** in the main event.
+  - If there are more than 70 teams participating in the Qualifier, the Group Stage will use the [split double-elimination format](/wiki/Tournaments/TMC/4th#Group-Stage-&-Playoffs).
+  - If there are fewer than 70 teams participating in the Qualifier, the Group Stage will use the standard double-elimination format.
+
+![Group format](/wiki/Tournaments/TMC/4th/image/group-stage-bracket.png)
+
+  - **Each team will have 2 bans in the group stage**, providing extra strategic options throughout the different weeks.
+- **A special series of "X" picks will apply for RC and LN slots**, giving players more options to pick harder charts.
+- Two teams from each group will advance, with a total of 32 teams competing in a standard double-elimination bracket for the championship.
+
+## Custom maps
+
+With nearly **50 custom mappers** contributing, this year we are bringing you mappools that are **almost entirely custom-made** with maps ranging from the lower end of the difficulty spectrum to the very top. We hope this will not only enhance our Touhou journey, but also contribute to the osu! community by adding more Touhou maps!
```

Comment:
```suggestion
With nearly **50 custom mappers** contributing, this year we are bringing you mappools that are **almost entirely custom-made** with maps ranging from the lower end of the difficulty spectrum to the very top. We hope this will not only enhance our Touhou journey but also contribute to the osu! community by adding more Touhou maps!
```